backend/danswer/server/manage/administrative.py

```diff
@@ -231,3 +235,56 @@ def update_token_budget_settings(
     # Store the settings in the dynamic config store
     get_dynamic_config_store().store(TOKEN_BUDGET_SETTINGS, settings_json)
     return {"message": "Token budget settings updated successfully."}
+
+
+@router.put("/admin/users")
+def bulk_invite_users(
+    _: User | None = Depends(current_admin_user),
+    emails: list[str] = Body(..., embed=True),
```

Comment:
super nit (doesn't matter): slight preference across the board to have the more important/variable arguments on top and the "less interesting" args under. So emails first, and the user dependency after